charts/virtual-kubelet/templates/deployment.yaml

```diff
@@ -15,18 +15,38 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         env:
         - name: AZURE_AUTH_LOCATION
-          value: /etc/virtual-kubelet/credentials.json
+          value: /etc/virtual-kubelet/auth/credentials.json
         - name: ACI_RESOURCE_GROUP
           value: {{ .Values.env.aciResourceGroup }}
         - name: ACI_REGION
           value: {{ default "westus" .Values.env.aciRegion }}
+        - name: APISERVER_CERT_LOCATION
```

Comment:
what's the requirement for this cert? how can the k8s admin generate the cert?

Reply:
`openssl req -newkey rsa:4096 -new -nodes -x509 -days 3650 -keyout key.pem -out cert.pem` I will add this logic to the chart

Reply:
Don't quite understand. is it a self-signed cert? looks like it's used in the TLS. how does the ApiServer trust this cert?